docs/compliance/0-Setup-AdministrativeAccountAndPermission.md

```diff
@@ -0,0 +1,82 @@
+# Automated Foundational Architecture for PCI DSS-Compliant Environments (v1)
```

Comment:
Rather than format these script topics like reference pages (with description, examples), I think we should use a procedural approach that shows how to deploy, with steps. See the SharePoint and SAP RAs for an example of what I mean.